docs/user-guide/story-setting.md

```diff
@@ -0,0 +1,19 @@
+ 
+ 
+For setting the information that appears on the story top bar on the *View Mode* the user need to go back on the *Edit Mode* and open the **Setting** button <img src="../img/button/setting-button.jpg" class="ms-docbutton"/>.
```

Comment:
The story settings panel allows the editor to configure which additional components should appear to end users in the story view (like the Title, a Logo and the Navigation Toolbar): these components, if properly configured, will appear in the top bar of the story itself. The Story settings panel is available in *Edit Mode* and can be opened by clicking on the **Setting** button <img src="../img/button/setting-button.jpg" class="ms-docbutton"/>.